plugins/hls-hlint-plugin/src/Ide/Plugin/Hlint.hs

```diff
@@ -125,7 +130,9 @@ rules plugin = do
     ideas <- if hlintOn' then getIdeas file else return (Right [])
     return (diagnostics file ideas, Just ())
 
-  getHlintSettingsRule (HlintEnabled [])
+  defineNoFile $ \GetHlintSettings -> do
+      (Config flags) <- getHlintConfig plugin
+      getHlintSettingsRule (HlintEnabled flags)
```

Comment:
Are we using `HlintDisabled` somewhere? If no, can we remove `HlintUsage`?

Reply:
I was also wondering this, I don't think it is used anywhere else.